packages/jetpack-connection/composer.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "automattic/jetpack-connection",
+	"description": "Everything needed to connect to the Jetpack infrastructure",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"version": "1.0.0",
+	"require": {},
+	"require-dev": {
+		"phpunit/phpunit": "7.*.*",
+		"10up/wp_mock": "0.4.2"
```

Comment:
The problem with WP Core tests is that it requires a working WordPress installation, whereas this approach allows us to really isolate the components and run unit tests as they were meant to be run. I don't see a problem with having separate testing frameworks as long as they don't get in our way.

Reply:
Thanks for elaborating. To be fair, I really love your idea ❤️

However, isn't it too far from us? Our current tests rely on having a working WordPress installation, and many of them actually would need more than just mocking one or two WordPress functions/classes. 

Ideally, our plugin and tests would work without WordPress, but we have to be fair that this seems a bit far from what we currently have. And also I think it would cause a challenge for migrating to independent packages, at least initially.

I wonder if we could and should postpone the tests migration to non-WP-Core based ones to another time, when we can focus better on it?

Reply:
Thanks for elaborating. To be fair, I really love your idea ❤️

However, isn't it too far from us? Our current tests rely on having a working WordPress installation, and many of them actually would need more than just mocking one or two WordPress functions/classes. 

Ideally, our plugin and tests would work without WordPress, but we have to be fair that this seems a bit far from what we currently have. And also I think it would cause a challenge for migrating to independent packages, at least initially.

I wonder if we could and should postpone the tests migration to non-WP-Core based ones to another time, when we can focus better on it?